pom.xml

```diff
@@ -1688,7 +1688,7 @@
             </build>
         </profile>
 
-        <profile>
+<!--        <profile>
```

Comment:
I confirmed that janusgraph-core completes with doclint enabled, but we can't re-enable doclint in the top pom.xml until all of the other janusgraph-* packages have their errors fixed.

Reply:
I removed the changes to pom.xml and added [skip ci] to prevent wasting ci cycles